integration/commands_administration_test.go

```diff
@@ -185,18 +185,69 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
 		databaseName: "admin",
 	})
 
-	var actual bson.D
-	err := collection.Database().RunCommand(ctx, bson.D{{"getParameter", "*"}}).Decode(&actual)
-	require.NoError(t, err)
+	for name, tc := range map[string]struct {
+		command    bson.D
+		expected   map[string]any
+		unexpected []string
+		err        *mongo.CommandError
+	}{
+		"AllParameters": {
+			command: bson.D{{"getParameter", "*"}},
+			expected: map[string]any{
+				"acceptApiVersion2": false,
+				"authSchemaVersion": int32(5),
+				"quiet":             false,
+				"ok":                float64(1),
+			},
+		},
+		"ExistingParameters": {
+			command: bson.D{{"getParameter", 1}, {"quiet", 1}},
```

Comment:
What would happen for `{"getParameter", 0}`? `{"getParameter", NULL}`? Should we add tests for those cases?

Reply:
fix